### ISSUE TITLE
Links Lack a Title on Hover

### ISSUE THREAD

**Issue by mckaymichael** [mckaymichael/portfolio#2]
## Status
Assigned

## Description
Not all links contain titles. This is an issue because those with accessibility issues will have a lot of trouble navigating my website. It may also show that I'm not willing to do the more tedious tasks that need to be done.

## Steps to Reproduce
1. [Go to my portfolio](https://michaelkmckay.com/).
2. Hover the mouse over any link except the "Projects" icon in the header.
3. Wait and you'll notice nothing happens.

## Expected Results
* All links should produce a title when hovered over.

## Actual Result
* Links do not provide a title on hover.

## Configuration
This bug was found on the following configurations:
 ...... | Safari | Chrome | Brave | Edge
--- | --- | --- | --- | ---
OS X 10.15 | found | found | found | found
Windows 11 | found | found | found | found

## Severity
Low

## Priority
Low

## Type:
Accessibility Bug

## Assignee:
Michael McKay
